src/api/pipes/lowercase-pipe.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Created by fmontes on 7/28/16.
+ */
+import {Pipe, PipeTransform} from '@angular/core';
+
+@Pipe({
+    name : 'lowercase',
+})
+
+/**
+ * Put in lowercase the received string
+ */
+export class LowercasePipe implements PipeTransform {
+
+    transform(value: string, args: Array<any>): string {
+        return value === undefined || value === '' ? '' : value.toLowerCase();
```

Comment:
I have seem things such as value === undefined || value === '' ? '' or similar, shouldn't we encapsulate them into a ValidationUtils class so can be reused around all the code.